server/controllers/users/deactivateUser.js

```diff
@@ -1,3 +1,16 @@
-exports.deactivateUser = (req, res) => {
-  res.send('Hi from deactivate file');
+const { users: { deactivateUser } } = require('../../database/queries');
+
+exports.deactivateUser = (req, res, next) => {
+  // Get the user id and username
+  // const { userId, username } = req.user;
```

Comment:
oops commented code :open_mouth: 
I know you will tell me that the auth not implemented for now but as you make we know the key 
` req.user`  so you don't need to comment it

Reply:
Because it gives errors in the terminal